bigquery/google/cloud/bigquery/job.py

```diff
@@ -918,6 +958,7 @@ class CopyJobConfig(object):
 
     def __init__(self):
         self._properties = {}
+        self._kms_key_name = None
```

Comment:
What's special about this that it doesn't go under `properties` like most everything else?

Reply:
Good point. Removed and refactored.